docs/data/data-grid/demo/PopularFeaturesDemo.js

```diff
@@ -186,6 +187,15 @@ export const featuresSet = [
     demo: <AddNewColumnMenuGrid />,
     newBadge: true,
   },
+  {
+    id: 19,
+    name: 'Clipboard paste',
+    description: 'Copy and paste data using the clipboard',
+    plan: 'Premium',
+    detailPage: '/clipboard/',
+    demo: <ClipboardPaste />,
```

Comment:
It's essentially a data grid with no visual indication of how to test the functionality (or is it obvious already by name/title 🤔). I am not sure if it's easily possible with this demo thing, but maybe adding copied rows' feedback like in [this demo](https://mui.com/x/react-data-grid/clipboard/#clipboard-copy) will add more engagement to this demo? 🤷‍♂️

Reply:
I've updated the demo description to make it more clear:

> Copy and paste the selected cells and rows using the copy/paste keyboard shortcuts.